index.bs

```diff
@@ -4890,9 +4890,9 @@ partial interface MLGraphBuilder {
   <div class=algorithm-steps>
     1. Let |outputShape| be an empty array of {{unsigned long}}.
     1. If |newShape|'s [=list/size=] is 0, set |outputShape| to an empty [=/list=] for a scalar.
-    1. If any value in |newShape| is 0, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
+    1. If any value in |newShape| is 0, then [=exception/throw=] a {{RangeError}}.
```

Comment:
FYI `RangeError` is used here, too, though since this is checking _the values within a list_ you could argue that `DataError` is more appropriate here (or `TypeError` I suppose). Our decision here may depend on what we decide for line 1219

Reply:
`RangeError` sounds good to me.